src/components/facets/FacetConnected.tsx

```diff
@@ -1,18 +1,19 @@
 import * as React from 'react';
-import { connect } from 'react-redux';
+import {connect} from 'react-redux';
 import * as _ from 'underscore';
-import { IReactVaporState, IReduxActionsPayload } from '../../ReactVapor';
-import { IReduxAction, ReduxUtils } from '../../utils/ReduxUtils';
-import { Facet, IFacet, IFacetDispatchProps, IFacetOwnProps, IFacetProps, IFacetStateProps } from './Facet';
-import { addFacet, changeFacet, emptyFacet, removeFacet } from './FacetActions';
-import { IFacetState } from './FacetReducers';
+import {IReactVaporState, IReduxActionsPayload} from '../../ReactVapor';
+import {IReduxAction, ReduxUtils} from '../../utils/ReduxUtils';
+import {Facet, IFacet, IFacetDispatchProps, IFacetOwnProps, IFacetProps, IFacetStateProps} from './Facet';
+import {addFacet, changeFacet, emptyFacet, removeFacet} from './FacetActions';
+import {IFacetState} from './FacetReducers';
 
 const mapStateToProps = (state: IReactVaporState, ownProps: IFacetOwnProps): IFacetStateProps => {
-  const item: IFacetState = _.findWhere(state.facets, { facet: ownProps.facet.name });
+  const item: IFacetState = _.findWhere(state.facets, {facet: ownProps.facet.name});
+  const selectedFacetRows = item ? ownProps.facetRows.filter((row) => _.findWhere(item.selected, {name: row.name})) : [];
```

Comment:
Gets the `selectedFacetRows` from the props using the name as an index. This way, if an info (for now the `count`) is modified from the `facetRows` prop, changes will be passed to the `selectedFacetRows`, before it was not the case.